### PR TITLE
Login improvements

### PR DESCRIPTION
**Pull request by ThomasS-kainos**
The login button is only present when the user is not logged in, and will change to logout when logged in.

### CODE REVIEW

**Review by AdamB-Kainos — APPROVED** [review]

File: src/services/JobRoleService.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
+import axios, { AxiosResponse } from "axios";
+import { JobRoleResponse } from "../main/models/JobRoleResponse";
```
Comment (boydstepkainos) [inline src/services/JobRoleService.ts]: What is this? Seems unrelated to ticket title?

File: views/snippets/nav.html [hunk 2]
```diff
@@ -5,8 +5,14 @@
             <a class="nav-item nav-link active" href="/">Home</span></a>
             <a class="nav-item nav-link" href="/job-roles">Jobs</span></a>
             <a class="nav-item nav-link" href="/">About Us</span></a>
-            <a class="nav-item nav-link" href="/loginForm">Login</span></a>
-            <a class="nav-item nav-link" href="/logoutForm">Logout</span></a>
+            <div class="right"></div>
+            {% if not token.length %}
```
Comment (boydstepkainos) [inline views/snippets/nav.html]: change to if else

Comment (boydstepkainos) [reply views/snippets/nav.html]: if else, elif is another if condition

Comment (ThomasS-kainos) [reply views/snippets/nav.html]: Just changed that to a If and Else statement

**Review by shaunganley — APPROVED** [review]
